ansibullbot/utils/file_tools.py

```diff
@@ -472,3 +473,22 @@ def isnewdir(self, path):
             return False
         else:
             return True
+
+    def commits_by_email(self, email):
```

Comment:
Should not be `ModuleIndexer.emails_cache` dict (key are email, values are GitHub ID (string)) used/updated instead of adding another mapping ?